src/_scss/pages/addData/_content.scss

```diff
@@ -18,6 +18,14 @@
                 }
             }
         }
+        .usa-da-submissiontype {
+            .usa-da-submissiontype-group {
```

Comment:
While I'm here, why is `submissiontype` one word?

Reply:
`.usa-da-datetype` above was one so i kept it consistent

Reply:
It's weird that it's one word when it's clearly 2 words. At least update the one we made. If you want to clean up you can do the other one, not required.